scripts/test.ipynb

```diff
@@ -230,11 +232,29 @@
    ],
    "source": [
     "for strategy in strategies:\n",
+    "    avg =[]\n",
+    "    model = CAT.model.IRTModel(**config)\n",
     "    model = CAT.model.NCDModel(**config)\n",
     "    model.init_model(test_data)\n",
-    "    model.adaptest_load(ckpt_path)\n",
+    "    if strategy.name == 'BOBCAT':\n",
+    "        model.adaptest_load_BOBCAT(ckpt_path,bobcat_policy_path)\n",
+    "    else:\n",
+    "        model.adaptest_load(ckpt_path)\n",
     "    test_data.reset()\n",
-    "    \n",
+    "    print(strategy.name)\n",
+    "    if strategy.name == 'NCAT':\n",
```

Comment:
If it is hard to unify all the strategies, I would recommend to write train/test examples for them in seperate scripts, such as `train_NCAT.ipynb`, `test_NCAT.ipynb`, `test_BOBCAT.ipynb`.